src/styles/foundation/colors.scss

```diff
@@ -76,11 +76,25 @@ Available options: #{available-names($color-palette-data)}
 
 @function color-multiply($foreground, $background: null) {
   @if $background == null {
-    // stylelint-disable-next-line color-no-hex
-    $background: #ffffff;
+    $background: rgb(255, 255, 255);
   }
 
-  @return $foreground * $background / 255;
+  $red: red($background) * red($foreground) / 255;
+  $green: green($background) * green($foreground) / 255;
+  $blue: blue($background) * blue($foreground) / 255;
+
+  $opacity: opacity($foreground);
+  $background-opacity: opacity($background);
+
+  // calculate opacity
+  $bm-red: $red * $opacity + red($background) * $background-opacity *
```

Comment:
What does the `$bm` prefix stand for?

Reply:
blend-mode, as I moched this logic from https://github.com/heygrady/scss-blend-modes/blob/master/stylesheets/_blend-modes.scss (along with some refactoring so to keep this a single function)

Reply:
Ha! That didn't jump out, can we rename these variables for clarity?

Reply:
Any suggestions other than $red and $red2?